test/WebJobs.Script.Tests/TestScripts/Node/HttpTriggerExpressApi/index.js

```diff
@@ -0,0 +1,20 @@
+﻿var util = require('util');
+
+module.exports = function (context, req) {
+    context.log('Node.js HttpTrigger function invoked.');
+
+    context.res.status(200)
+        .set('test-header', 'Test Response Header')
+        .set('Content-Type', 'application/json; charset=utf-8')
+        .send({
```

Comment:
I think that tying send & end to context.done is the right move, it will match the express model.  Potential negative is if there are multiple output bindings, could get a little confusing.